src/app/config.ts

```diff
@@ -104,6 +124,7 @@ export class AltairConfig {
     this.initialData.query = (window as any).__ALTAIR_INITIAL_QUERY__ || initialQuery || '';
     this.initialData.variables = (window as any).__ALTAIR_INITIAL_VARIABLES__ || initialVariables || '';
     this.initialData.headers = (window as any).__ALTAIR_INITIAL_HEADERS__ || initialHeaders || '';
+    this.initialData.environments = (window as any).__ALTAIR_INITIAL_ENVIRONMENTS__ || initialEnvironments || {};
```

Comment:
Since this is a new configuration, we shouldn't add the *`__ALTAIR_INITIAL_ENVIRONMENTS__`* option. We are deprecating that way of setting the config. Instead, they should be passed to `AltairGraphQL.init()`.

Reply:
Thanks I’ll update this. Do you have docs or an example of this style of configuration in use?

Reply:
You are already using it, with `initialEnvironments`. You can just take out the `window.__ALTAIR_INITIAL_ENVIRONMENTS__` part

Reply:
Updated in d0be177